db/structure.sql

```diff
@@ -4219,748 +4213,6 @@ CREATE MATERIALIZED VIEW public.reporting_facility_states AS
   WITH NO DATA;
 
 
---
--- Name: MATERIALIZED VIEW reporting_facility_states; Type: COMMENT; Schema: public; Owner: -
---
-
-COMMENT ON MATERIALIZED VIEW public.reporting_facility_states IS 'Monthly summary of a facility''s indicators. This table has one row per facility, per month, from the month of the facility''s first registration.';
-
-
---
--- Name: COLUMN reporting_facility_states.month_date; Type: COMMENT; Schema: public; Owner: -
```

Comment:
All these comments removed in this file are necessary yeah? Maybe out of date with the migrations on master?

Reply:
We should run `db:refresh_matviews` after running `db:migrate` to add comments to the table columns. 
Can you please run `db:refresh_matviews and add back those comments?

Reply:
I've added back those comments by running db:migrate.